tests/cases/06_gridfs.ts

```diff
@@ -1,9 +1,17 @@
 import { GridFSBucket } from "../../mod.ts";
 import { testWithClient } from "../common.ts";
-import { assert, assertEquals, bytesEquals } from "../test.deps.ts";
-
-function bufferEquals(a: ArrayBuffer, b: ArrayBuffer) {
-  return bytesEquals(new Uint8Array(a), new Uint8Array(b));
+import {
+  assert,
+  assertEquals,
+  assertNotEquals,
+  readAll,
+  readerFromStreamReader,
+} from "../test.deps.ts";
+
+async function streamReadAll(readable: ReadableStream): Promise<Uint8Array> {
+  const reader = readerFromStreamReader(readable.getReader());
```

Comment:
Please make this to a Response. Else we would need to delete this code at Deno 2.0 release

Reply:
Is it the best option? `Response` is a part of fetch and fetch is an http client. why we should use something related to fetch here

Reply:
I'm also thinking about refactor gridfs from `ReadableStream/WritableStream` to `Reader/Writer`. because `Reader/Writer` can be used with web APIs like fetch easily (without using std/stream utils), but `ReadableStream/WritableStream` requires std/stream to be converted into `Reader/Writer`

Reply:
Thats stupid. really. Deno wants to Remove these.

Reply:
Reader and Writer is incompatible with the Web. Deno itself switches everything away from the old go still streams.

Why should we go against that?

Reply:
Ok, let's merge this PR, I'll remove this Reader/Writer usage in an another PR